### PR TITLE
Alerts v2

### PR DESCRIPTION
**Pull request by mirii1994**
This PR adds compatibility to logzio's new Alerts API (v2)

### CODE REVIEW

**Review by yyyogev — APPROVED** [review]
Superb! 🔥

File: alerts_v2/client_alerts_v2.go [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+type AlertOutput struct {
+	Recipients AlertRecipients `json:"recipients,omitempty"`
+	SuppressNotificationsMinutes int `json:"suppressNotificationsMinutes,omitempty"`
+	Type string `json:"type,omitempty"`
+}
+
+type AlertRecipients struct {
+	Emails []string `json:"emails,omitempty"`
+	NotificationEndpointIds []int `json:"notificationEndpointIds,omitempty"`
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply alerts_v2/client_alerts_v2.go:68]: This structure is also being used for the Update action, so if I won't add `omitempty` it'll cause problems for that action

File: alerts_v2/client_alerts_v2_delete.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	}
+
+	jsonBytes, _ := ioutil.ReadAll(resp.Body)
+
+	if !logzio_client.CheckValidStatus(resp, []int{deleteAlertMethodSuccess}) {
+		return fmt.Errorf("API call %s failed with status code %d, data: %s", "DeleteAlert", resp.StatusCode, jsonBytes)
+	}
+
+	str := fmt.Sprintf("%s", jsonBytes)
+	if strings.Contains(str, fmt.Sprintf("alert id %d not found", alertId)) {
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply alerts_v2/client_alerts_v2_delete.go]: You're absolutely right about both of your remarks. I removed the redundant condition in all the operations it appeared on :)

File: utils.go [hunk 3]
```diff
@@ -56,3 +57,18 @@ func CreateHttpRequest(req *http.Request) (map[string]interface{}, error) {
...
+		return nil, err
+	}
+	defer resp.Body.Close()
+	jsonBytes, err := ioutil.ReadAll(resp.Body)
+	if !CheckValidStatus(resp, []int{serviceSuccess, serviceNoContest, serviceCreated}) {
+		return nil, fmt.Errorf("%d %s", resp.StatusCode, jsonBytes)
+	}
+
+	return jsonBytes, nil
+}
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply utils.go:65]: Done

File: alerts_v2/client_alerts_v2_delete.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (yyyogev) [reply alerts_v2/client_alerts_v2_delete.go]: Uhh.. I meant that this should be an `else` clause for the `checkValidStatus` condition, not remove it..

**Review by yyyogev — APPROVED** [review]
Perfect! 💯

**Pull request merged**